include/uv-win.h

```diff
@@ -366,8 +366,8 @@ RB_HEAD(uv_timer_tree_s, uv_timer_s);
     struct {                                                                  \
       OVERLAPPED overlapped;                                                  \
       size_t queued_bytes;                                                    \
-    };                                                                        \
-  };                                                                          \
+    } io;                                                                     \
+  } req;                                                                      \
```

Comment:
@bnoordhuis @txdv do you guys have any suggestion for this one? Maybe "u"? (I think I've ssen u used somewhere, maybe not libuv)